numba/np/arrayobj.py

```diff
@@ -12,7 +12,7 @@
 
 import numpy as np
 
-from numba import pndindex, literal_unroll
+from numba import pndindex, literal_unroll, cuda, njit, types #type : ignore
```

Comment:
`types` was already imported. `cuda` cannot be imported in Numba outside of the `numba.cuda` module in general.
```suggestion
from numba import pndindex, literal_unroll
```